contact/index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-Uh-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="/style.scss" />
+    <link rel="stylesheet" href="/contact/style.scss" />
+    <script type="text/javascript" src="/contact/contact-script.js"></script>
```

Comment:
```suggestion
    <script src="/contact/contact-script.js"></script>
```
`type="text/javascript"` は無くても問題ありません。